integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -1257,6 +1402,27 @@ private void scalingClusters(String clusterName, String serverPodName, int repli
     logger.info("The cluster {0} scaled successfully.", clusterName);
   }
 
+  private void restoreEnv() {
+    int newReplicaCount = 2;
+    String configServerName = "config-cluster-server" + newReplicaCount;
+    String configServerPodName = domainUid + "-" + configServerName;
+    String dynamicServerName = "managed-server" + newReplicaCount;
+    String dynamicServerPodName = domainUid + "-" + dynamicServerName;
+
+    // restore test env
+    assertDoesNotThrow(() ->
+        executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, configServerName),
+        String.format("Failed to run %s", STOP_SERVER_SCRIPT));
+    checkPodDeleted(configServerPodName, domainUid, domainNamespace);
+    logger.info("managed server " + configServerPodName + " stopped successfully.");
+
+    assertDoesNotThrow(() ->
+        executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, dynamicServerName),
+        String.format("Failed to run %s", STOP_SERVER_SCRIPT));
+    checkPodDeleted(dynamicServerPodName, domainUid, domainNamespace);
+    logger.info("managed server " + dynamicServerPodName + " stopped successfully.");
+  }
+
   private static void createDomainSecret(String secretName, String username, String password, String domNamespace) {
```

Comment:
Use the utility method oracle.weblogic.kubernetes.utils.CommonMiiTestUtils.createDomainSecret

Reply:
Changed